### PR TITLE
update example to 1.8.0

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

**Comment by SandyEma9** [comment]
@iulianatodoran @robert-altom @RobertPoienar @gomboskriszta15

File: requirements.txt [hunk 1]
```diff
@@ -3,6 +3,6 @@ pytest
 pytest-selenium
 virtualenv
 Appium-Python-Client
-altunityrunner==1.6.1
+alttester==1.8.0
```
Comment (robert-altom) [inline requirements.txt:6]: Here it needs to be `AltTester-Driver`.

File: pages/game_play_page.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        self.pause_button.tap()
+
+    def get_current_life(self):
+        return int(self.character.call_component_method('CharacterInputController', 'get_currentLife', assembly=''))
+
+    def jump(self):
+        self.character.call_component_method('CharacterInputController', 'Jump', assembly='')
+
+    def move_right(self):
+        self.character.call_component_method('CharacterInputController', 'ChangeLane', parameters=['1'])
```
Comment (robert-altom) [inline pages/game_play_page.py:36]: For all calls of `call_component_method` you should be updated from: 

```python 
call_component_method('CharacterInputController', 'get_currentLife', assembly='')
call_component_method('CharacterInputController', 'ChangeLane', parameters=['1'])
```

To: 

```python 
call_component_method('CharacterInputController', 'get_currentLife', '')
call_component_method('CharacterInputController', 'ChangeLane', '', parameters=['1'])
```

**Comment by SandyEma9** [comment]
@robert-altom @RobertPoienar @gomboskriszta15 @iulianatodoran

**Comment by SandyEma9** [comment]
Need to put manual the apk for 1.8.0

File: requirements.txt [hunk 3]
```diff
@@ -3,6 +3,6 @@ pytest
 pytest-selenium
```
Comment (robert-altom) [inline requirements.txt:3]: In `requirements.txt` you only need: 

```
pytest
Appium-Python-Client
AltTester-Driver==1.8.0

```

**Comment by iulianatodoran** [comment]
The GitLab link in the README.md should be updated with the GitHub one.

**Review by iulianatodoran — APPROVED** [review]
LGTM

**Review by RobertPoienar — APPROVED** [review]
LGTM

**Pull request merged**